libs/ledgerjs/package.json

```diff
@@ -3,19 +3,17 @@
   "private": true,
   "version": "0.1.2",
   "scripts": {
-    "test:importer": "jest script/crypto-assets-importer/",
-    "import-cal-tokens": "ts-node script/crypto-assets-importer/index.ts"
+    "test:importer": "jest ./packages/cryptoassets/src/crypto-assets-importer/",
+    "import:cal-tokens": "ts-node ./packages/cryptoassets/src/crypto-assets-importer/index.ts"
```

Comment:
Why not a script anymore?
Wouldn't it belong in `./packages/cryptoassets/script/`?

Reply:
Because now as it's part of ledgerjs we can import `cryptocurrenciesById` in the evm importer